data_sources/mavedb/gene_mapping.py

```diff
@@ -0,0 +1,602 @@
+    "TRIM49C B-box-type zinc finger domain": "TRIM49C",
+    "TRIM6 B-box-type zinc finger domain": "TRIM6",
+    "TRIM71 B-box-type zinc finger domain": "TRIM71",
+    "TRMT13 Zinc finger, CCCH-type, TRM13 domain": "TRMT13",
+    "TTC3 Zinc finger, RING-type domain": "TTC3",
+    "UBE2K Ubiquitin-associated domain": "UBE2K",
+    "UBE3A Ubiquitin-protein ligase E3A, N-terminal zinc-binding domain": "UBE3A",
+    "UBE4B U box domain": "UBE4B",
+    "UBQLN3 Ubiquitin-associated domain": "UBQLN3",
+    "UBTF High mobility group box domain": "UBTF",
+    "UBXN2B UBX domain": "UBXN2B",
+    "UFM1 Ubiquitin-fold modifier 1": "UFM1",
+    "USH1C PDZ domain": "USH1C",
+    "VAV1 SH3 domain": "VAV1",
+    "VAX1 Homeobox domain": "VAX1",
+    "VCP CDC48, N-terminal subdomain": "VCP",
+    "VIL1 Villin headpiece domain": "VIL1",
+    "WBP4 U1-C, C2H2-type zinc finger domain": "WBP4",
+    "WNK1 Serine/threonine-protein kinase OSR1/WNK, CCT domain": "WNK1",
+    "WSB2 SOCS box domain": "WSB2",
+    "WWP1 WW domain": "WWP1",
+    "WWP2 WW domain": "WWP2",
+    "WWTR1 WW domain": "WWTR1",
+    "XRCC6 SAP domain": "XRCC6",
+    "YAP1 WW domain": "YAP1",
+    "YBX1 Cold-shock protein, DNA-binding domain": "YBX1",
+    "YBX2 Cold-shock protein, DNA-binding domain": "YBX2",
+    "YBX3 Cold-shock protein, DNA-binding domain": "YBX3",
+    "YY1 Zinc finger C2H2-type domain": "YY1",
+    "ZBTB20 Zinc finger C2H2-type domain": "ZBTB20",
+    "ZBTB24 Zinc finger C2H2-type domain": "ZBTB24",
+    "ZC3H10 Zinc finger, CCCH-type domain": "ZC3H10",
+    "ZC3H15 E3 ligase, CCCH-type zinc finger domain": "ZC3H15",
+    "ZC3H7A Zinc finger, CCCH-type domain": "ZC3H7A",
+    "ZEB1 Zinc finger C2H2-type domain": "ZEB1",
+    "ZFC3H1 Putative zinc-finger domain": "ZFC3H1",
+    "ZFHX2 Homeobox domain": "ZFHX2",
+    "ZFHX3 Homeobox domain": "ZFHX3",
+    "ZFHX4 Homeobox domain": "ZFHX4",
+    "ZFP30 Zinc finger C2H2-type domain": "ZFP30",
+    "ZFP36L2 Zinc finger, CCCH-type domain": "ZFP36L2",
+    "ZFP36 Zinc finger, CCCH-type domain": "ZFP36",
+    "ZFP57 Zinc finger C2H2-type domain": "ZFP57",
+    "ZFYVE16 FYVE zinc finger domain": "ZFYVE16",
+    "ZHX1 ZHX, C2H2 finger domain": "ZHX1",
+    "ZIC2 ZIC protein, zinc finger domain": "ZIC2",
+    "ZIC3 Zinc finger C2H2-type domain": "ZIC3",
+    "ZMAT3 Zinc finger, double-stranded RNA binding domain": "ZMAT3",
+    "ZMYM2 Zinc finger, MYM-type domain": "ZMYM2",
+    "ZMYM3 Zinc finger, MYM-type domain": "ZMYM3",
+    "ZMYM5 Zinc finger, MYM-type domain": "ZMYM5",
+    "ZMYM6 Zinc finger, MYM-type domain": "ZMYM6",
+    "ZNF148 Zinc finger C2H2-type domain": "ZNF148",
+    "ZNF292 Zinc finger C2H2-type domain": "ZNF292",
+    "ZNF408 Zinc finger C2H2-type domain": "ZNF408",
+    "ZNF711 Zinc finger C2H2-type domain": "ZNF711",
+    "ZNF81 Zinc finger C2H2-type domain": "ZNF81",
+    "ZRANB2 Zinc finger, RanBP2-type domain": "ZRANB2",
+    "ZYX Zinc finger, LIM-type domain": "ZYX",
+    "ZZZ3 SANT/Myb domain": "ZZZ3",
+}
```

Comment:
Would it be better to keep this mapping in ES and then get it dynamically?

Reply:
This is up for discussion, several points in play here:

On the one hand, I think that if anything can be precomputed and served statically, it should be: makes things faster and reduces server load.

On the other hand, once we start adding more datasources, we will almost certainly completely overhaul the way genes & their mappings are handled anyway. For one, the (normalised) gene will probably be the central grouping point for the data, and in that case, we will indeed store it there.

I suggest leavings things as they are for now, and we can rethink the approach after the release, if that sounds all right